tests/local.rs

```diff
@@ -423,9 +423,13 @@ async fn timeouts(c: Client, _: u16) -> Result<(), error::CmdError> {
 async fn dynamic_commands(c: Client, port: u16) -> Result<(), error::CmdError> {
     let sample_url = sample_page_url(port);
     c.goto(&sample_url).await?;
-    let title = c.issue_cmd(WebDriverCommand::GetTitle).await?;
+    let title = c
+        .issue_cmd(test_wrap_command(WebDriverCommand::GetTitle))
+        .await?;
     assert_eq!(title.as_str(), Some("Sample Page"));
-    let title = c.issue_cmd(Box::new(WebDriverCommand::GetTitle)).await?;
+    let title = c
+        .issue_cmd(test_wrap_command(WebDriverCommand::GetTitle))
+        .await?;
     assert_eq!(title.as_str(), Some("Sample Page"));
     Ok(())
 }
```

Comment:
I think what we're really looking to demonstrate here is that users can provide their own custom commands via fantoccini. That command doesn't even have to come from the `webdriver` crate. So how about we do this:
```rust
struct GetTitle;
impl WebDriverCompatibleCommand for GetTitle {
  // ...
}
```
and then check that it's possible to pass `GetTitle` and `Box<GetTitle>` to `issue_cmd`? Then you can get rid of `test_wrap_command` entirely (and the test helper feature).

Reply:
Thanks, this change removes the need for `test_wrap_command` and demonstrates `issue_cmd` can accept both `boxed` and `non-boxed` types.